src/build.ts

```diff
@@ -178,24 +157,32 @@ async function emitDom() {
     for (const [key, value] of Object.entries(descriptions)) {
       const target = idl.interfaces!.interface[key] || namespaces[key];
       if (target) {
-        target.deprecated = transformVerbosity(key, value);
+        target.deprecated = transformVerbosity(value);
       }
     }
     return idl;
   }
 
-  function transformVerbosity(name: string, description: string): string {
-    for (const regTemplate of removeVerboseIntroductions) {
-      const [{ source: template }, replace] = regTemplate;
-
-      const reg = new RegExp(template.replace(/\$\{name\}/g, name) + "\\s*");
-      const product = description.replace(reg, replace);
-      if (product !== description) {
-        return product.charAt(0).toUpperCase() + product.slice(1);
-      }
-    }
-
-    return description;
+  function transformVerbosity(description: string): string {
```

Comment:
We should move all MDN comment generation code to `mdn-comments.ts`. IMO this can be merged right into `extractSummary()`.